keanu-project/src/main/java/io/improbable/keanu/templating/SequenceItem.java

```diff
@@ -20,9 +20,19 @@
     private static Pattern NAME_REGEX = Pattern.compile(NAME_PREFIX + "-?[\\d]+$");
 
     private Map<VertexLabel, Vertex<?>> contents;
+    private int itemPosition;
+    private int uniqueIdentifier;
```

Comment:
better to call it `sequenceIdentifier` so it's clear that it's unique to the Sequence rather than the SequenceItem.